lib/sdf-server/src/service/v2/change_set/force_apply.rs

```diff
@@ -18,11 +19,23 @@ pub async fn force_apply(
     let mut ctx = builder
         .build(request_ctx.build(change_set_id.into()))
         .await?;
-
+    let change_set = ChangeSet::find(&ctx, change_set_id)
+        .await?
+        .ok_or(Error::ChangeSetNotFound(ctx.change_set_id()))?;
+    let old_status = change_set.status;
     ChangeSet::prepare_for_force_apply(&ctx).await?;
-
+    ctx.write_audit_log(
+        AuditLogKind::ApproveChangeSetApply {
+            from_status: old_status.into(),
+        },
+        change_set.name,
+    )
+    .await?;
     // We need to run a commit before apply so changes get saved
     ctx.commit().await?;
+    let change_set = ChangeSet::find(&ctx, ctx.visibility().change_set_id)
+        .await?
+        .ok_or(Error::ChangeSetNotFound(ctx.change_set_id()))?;
```

Comment:
I'm having a hard time understanding why we make this call, then apply to base, then call it again after posthog tracking. Is it possible this line moved down?

Reply:
sorted!